cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/0-intro.html

```diff
@@ -2,8 +2,18 @@
     {% if return_user %}
         <div class="content_wrapper">
             <div class="content_main">
+                <div class="u-hide-if-js">
+                    {% import 'v1/includes/molecules/notification.html' as notification with context %}
+                      {{ notification.render(
+                        'error',
+                        true,
+                        'This tool is not supported in your browser.
+                        Please try a newer browser or confirm that JavaScript is enabled.'
+                      ) }}
+                  </div>
```

Comment:
Mind breaking this out into a `no_js.html` template, and then some time in the future we'll maybe combine it together with the existing templates of this type. 

(for reference, we have these two [1](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/jinja2/owning-a-home/_templates/no_js.html), [2](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/jinja2/rural-or-underserved/_templates/no_js.html))

Reply:
Done 🎉